app/controllers/api/v1/timelines/home_controller.rb

```diff
@@ -44,7 +48,7 @@ def insert_pagination_headers
   end
 
   def pagination_params(core_params)
-    params.slice(:local, :limit).permit(:local, :limit).merge(core_params)
+    params.slice(:local, :limit, :no_bots).permit(:local, :limit, :no_bots).merge(core_params)
```

Comment:
Minor nitpick but if we have "exclude_replies" elsewhere it should be "exclude_bots" as well